### DISCUSSION TITLE
ApplicationService.GuidGenerator for multiple databases Create () is incorrect

### DISCUSSION THREAD

**Discussion by NickCc6** [abpframework/abp#21565] [Q&A]
I use  MySql and SQLServer
﻿
I first used UseMySQL and then UseSQLServer，GuidGenerator.Create()  data is SequentialGuidType.SequentialAsString 
﻿
But SQLServer is SequentialGuidType.SequentialAtEnd is more reasonable
﻿
Now，I Create Method CreateGuid(this IGuidGenerator guidGenerator, SequentialGuidType guidType) to work
﻿
Is there any good solution?

**Comment by maliming** [comment]
```cs
Configure<AbpSequentialGuidGeneratorOptions>(options =>
{
    options.DefaultSequentialGuidType = SequentialGuidType.SequentialAtEnd;
});
```

**Reply by NickCc6** [reply]
Hi
Maybe I didn't explain my question clearly

I think I Need SequentialGuidGenerator.Create(SequentialGuidType guidType)
Or is there any other better way


![image](https://github.com/user-attachments/assets/ae48b103-5d89-4f1c-870f-02890d7cd3ff)

**Comment by NickCc6** [comment]
I have implemented it in other ways


![image](https://github.com/user-attachments/assets/0b2c05a2-1389-4a51-8c19-99455346292e)
![image](https://github.com/user-attachments/assets/49124ee2-856b-4078-990c-81f2a31e396a)
